projects/content-data-admin/docker-compose.yml

```diff
@@ -26,3 +26,19 @@ services:
     environment:
       DATABASE_URL: "postgresql://postgres@postgres-9.6/content-data-admin"
       TEST_DATABASE_URL: "postgresql://postgres@postgres-9.6/content-data-admin-test"
+      DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: "true"
+
+  content-data-admin-app: &content-data-admin-app
+    <<: *content-data-admin
+    depends_on:
+      - content-data-api-app
+      - nginx-proxy
+      - postgres-9.6
+    expose:
+      - "3000"
+    command: bin/rails s --restart
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres-9.6/content-data-admin"
+      VIRTUAL_HOST: content-data-admin.dev.gov.uk
+      HOST: 0.0.0.0
```

Comment:
```suggestion
```

We only need `BINDING` here, since Rails 6.

Reply:
Thanks! Fixed up to keep commits tidier.